DOCS-1804-Add-RN-entries (#993)
tigera/docs: calico-enterprise_versioned_docs/version-3.18/_includes/release-notes/_v3.18.0-release-notes.mdx

### What's new {#whats-new-3.18.0}

#### Support for Azure CNI with overlay networking

> {{prodname}} adds supports for Azure CNI with overlay networking. This options augments support for traditional Azure CNI where a VNET IP address is assigned to every pod, which can lead to IP address exhaustion. To review both options, see [Microsoft Azure Kubernetes Service (AKS)](../getting-started/install-on-clusters/aks).

Reviewer: ```suggestion
{{prodname}} adds supports for Azure CNI with overlay networking. This options augments support for traditional Azure CNI where a VNET IP address is assigned to every pod, which can lead to IP address exhaustion.

For more information, see [the installation guide for Azure Kubernetes Service](../getting-started/install-on-clusters/aks).
```
A suggestion. Idea being that the title alone lacks sufficient context.

Reviewer: Location for this is TBD, pending plan for new versioning scheme.
